cmd/bcpd/app/codec.proto

```diff
@@ -17,13 +18,17 @@ message Tx {
     // escrow actions
     escrow.CreateEscrowMsg create_escrow_msg = 4;
     escrow.ReleaseEscrowMsg release_escrow_msg = 5;
-    escrow.ReturnEscrowMsg return_escrow_msg = 6;
+    escrow.ReturnEscrowMsg return_escrow_msg = 6; 
```

Comment:
What are you doing adding a space?
Please add "no-trailing-whitespace" to your editor.
Or better yet, set `gofmt` to run on the files upon save.

What editor do you use?